### ISSUE TITLE
ng lint with error because e2e/tsconfig.json not exist

### ISSUE THREAD

**Issue by bastienmoulia** [briebug/cypress-schematic#38]
**Describe the bug**
When i try to run `npm run lint` which is equivalent to `ng lint` i have an error because the *e2e/tsconfig.json* file is missing.
I think there is an oversight of renaming this file path.

**To Reproduce**
1. Create a new project with angular 9
2. Run `ng add @briebug/cypress-schematic`
3. Select **yes** to remove protractor from the project
4. Run `npm run lint`
5. See error :
```
An unhandled exception occurred: error TS5058: The specified path does not exist: 'my-project/e2e/tsconfig.json'.
```

**Comment by JonasStn** [comment]
if you deleted protractor you will have to delete the specific lint entry from angular.json

 ` "lint": {
          "builder": "@angular-devkit/build-angular:tslint",
          "options": {
            "tsConfig": [
              "tsconfig.app.json",
              "tsconfig.spec.json",
              "e2e/tsconfig.json"<-----
            ],
            "exclude": [
              "**/node_modules/**"
            ]
          }
        },`

**Comment by bastienmoulia** [comment]
IMHO the schematic should do that automatically.

**Comment by anthonymjones** [comment]
@bastienmoulia - Thank you for opening this issue, and I completely agree, the schematic should remove the `e2e/tsconfig.json` from the lint command automatically. If you're interested in opening a PR with the fix for this, you're more than welcome to, otherwise, I'm happy to take care of it when I have some time. Let me know!

**Comment by bastienmoulia** [comment]
I haven’t work with schematics, but I can give it a try this weekend.

**Comment by anthonymjones** [comment]
Closing, should be fixed in v4.0.3 release
